Sources/SourceParsingFramework/Utilities/ASTUtils.swift

```diff
@@ -41,6 +56,29 @@ public extension Structure {
         return dictionary["key.name"] as! String
     }
 
+    /// The type of this structure.
+    public var type: ASTType {
+        if let kindString = dictionary["key.kind"] as? String {
+            switch kindString {
+            case "source.lang.swift.decl.class": return .class
```

Comment:
nit: we can avoid typing the strings here by using the ones already in SourceKittenFramework `SwiftDeclarationKind.class.rawValue`

Reply:
Good call. I'll update this.